po/POTFILES

```diff
@@ -0,0 +1,13 @@
+AudioCutter/application.py
+AudioCutter/const.py
+AudioCutter/widgets/about.py
+AudioCutter/widgets/actionbar.py
```

Comment:
Peut-être rajouter l'appdata & le desktop/gschema aussi?

Reply:
oh, that why it was the app id?